### ISSUE TITLE
how to train the espvo model?

### ISSUE THREAD

**Issue by Bryan-bai** [ildoonet/deepvo#6]
Hi,
1.Can you present the espvo model training process?

2.In se3.py file line 27

        qw = tf.sqrt(tf.reduce_sum(tf.matrix_diag_part(matrix_rt), axis=-1)) / 2.0 

I think qw= tf.sqrt((tf.reduce_sum(tf.matrix_diag_part(matrix_rt), axis=-1)+1))/2.0, am I right?

thanks!

**Comment by OscarMind** [comment]
will you present an training process demo? thx

**Comment by Bryan-bai** [comment]
Hi,
I have a problem as #4  when I train this model, I have not sovled this problem.
Do you have this problem?
